substrate/frame/scheduler/src/lib.rs

```diff
@@ -1435,6 +1435,8 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
+// TODO: migrate `schedule::v2::Anon` to `v3`
```

Comment:
```suggestion
```
I know that there were some TODOs in the code before, but its not really good.  
Will create another issue to remove these traits in the future.

Reply:
Gotcha!